.github/workflows/sf-symbols.yml

```diff
@@ -0,0 +1,23 @@
+name: Update SF Symbols
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+
+    steps:
+    - uses: actions/github-script@v6
+      with:
+        github-token: ${{ secrets.LEO_SF_SYMBOLS_ACCESS_TOKEN }}
```

Comment:
Don't really like tokens, but I don't think we can trigger another repo without it.
cc @mihaiplesa to check and maybe create the token as I don't have permission on that repo.